### ISSUE TITLE
How can I preprocess "the coordinates should be scaled [0, 1]" ?

### ISSUE THREAD

**Issue by caihaunqai** [eriklindernoren/PyTorch-YOLOv3#688]
Hello, I  want to use custom dataset. However i got map 0.  I see "The coordinates should be scaled [0, 1]" in ReadMe.md.  How can I understand it? Can you help me to preprocess the coordinates ? Thank you!

**Comment by Flova** [comment]
The coordination should be expressed relative to the image dimensions meaning x and width of a bounding box are expressed as a fraction of the image width and y and height as a fraction of the image height.

Example:
Our bounding box is 400 px wide and our image is 800px wide.
Resulting in the value 0.5 for the width in the label file.

**Comment by caihaunqai** [comment]
Thank you very much.I follow what you said and it works. However, I got map 0.00146 for 35 epoches. Do you think it is normal? I don't use pretrained model. If it's wrong, do i miss anything?&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "eriklindernoren/PyTorch-YOLOv3"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月14日(星期五) 下午3:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [eriklindernoren/PyTorch-YOLOv3] How can I preprocess "the coordinates should be scaled [0, 1]" ? (#688)





 
The coordination should be expressed relative to the image dimensions meaning x and width of a bounding box are expressed as a fraction of the image width and y and height as a fraction of the image height.
 
Example:
 Our bounding box is 400 px wide and our image is 800px wide.
 Resulting in the value 0.5 for the width in the label file.

**Comment by Flova** [comment]
Depending on your dataset and the fact that you don't use the pre-trained weights it could be alright. 
How large is your dataset?

**Comment by caihaunqai** [comment]
About 14000 images. 12000 for train. Do you think how many epoches will have a good results? It's my first training. Thank you~




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "eriklindernoren/PyTorch-YOLOv3"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月14日(星期五) 下午3:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [eriklindernoren/PyTorch-YOLOv3] How can I preprocess "the coordinates should be scaled [0, 1]" ? (#688)





 
Depending on your dataset and the fact that you don't use the pre-trained weights it could be alright.
 How large is your dataset?

**Comment by Flova** [comment]
Without pre-trained weights about 300 or so are possible. It also may be harder to get it to converge without it.

**Comment by caihaunqai** [comment]
Thank you very much. I will try to use pretrained model!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "eriklindernoren/PyTorch-YOLOv3"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月14日(星期五) 下午4:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [eriklindernoren/PyTorch-YOLOv3] How can I preprocess "the coordinates should be scaled [0, 1]" ? (#688)





 
Without pre-trained weights about 300 or so are possible. It also may be harder to get it to converge without it.
